asistente_ladm_col/core/reports/base_report_generator.py

```diff
@@ -292,6 +292,8 @@ def generate_report(self, output_folder):
                     self.logger.debug(__name__, "{}:{}".format(plot_id, proc.exitCode()))
                     if proc.exitCode() == 0:
                         count += 1
+                    else:
+                        os.remove(current_report_path)  # Deletes the report when it could not be generated correctly
```

Comment:
Next time, pass a `os.remove()` inside a try except clause. We never know...

Reply:
Thanks for the feedback. I will keep it in mind

Reply:
We should also add the check here:

https://github.com/SwissTierrasColombia/Asistente-LADM-COL/blob/0ca0e173afdf1eb55ee1fa98bd233f36368c8b0b/asistente_ladm_col/core/reports/base_report_generator.py#L304

Reply:
So it is a configuration file that we generate ourselves to run the report, it may not be necessary to add the validation. I am attentive if you would like me to make the adjustment. Thank you

Reply:
> We should also add the check here:
> 
> https://github.com/SwissTierrasColombia/Asistente-LADM-COL/blob/0ca0e173afdf1eb55ee1fa98bd233f36368c8b0b/asistente_ladm_col/core/reports/base_report_generator.py#L304

Yes, but that's in a folder we know we can write/delete, whereas reports are generated in a folder given by the user.